modules/datadict/jsx/columnFormatter.js

```diff
@@ -12,8 +12,8 @@ function formatDataDictColumn(column, cell, rowData, rowHeaders) {
   if (loris.hiddenHeaders.indexOf(column) > -1) {
     return null;
   }
-
-  if (column === 'Description') {
+  const hasEditPermission = loris.userHasPermission('data_dict_edit');
+  if (column === 'Description' && hasEditPermission === true) {
```

Comment:
Not a big deal but I think `hasEditPermission` is a descriptive enoguh name that you don't need `=== true`

`if (column === 'Description' && hasEditPermission) {` is a bit cleaner